javascript/jwt-hardcode/jwt-hardcode.js

```diff
@@ -1,5 +1,5 @@
 "use strict";
-// todoruleid: hardcoded-jwt-secret
+// ruleid: hardcoded-jwt-secret
```

Comment:
any of patterns fail to fire on this line because, i think you specifically have https://sgrep.live/49z
```
var $JWT = require("jsonwebtoken");
...
$JWT.sign($P, "...", ...);
```
so without knowing the type of variable `obj` is string literal, your check wont fire here.